flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -53,7 +53,21 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table-api-java</artifactId>
+			<artifactId>flink-hadoop-compatibility_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-planner-blink_${scala.binary.version}</artifactId>
```

Comment:
Have you discussed about this? IMO the blink planner should not be dependent.

Reply:
Yes we have. I thought the blink planner will eventually replace flink's. Is there any reason why we can't depend on it?

Reply:
Firstly, this will not happen very soon. Secondly, i don't see a reason that a planner should be dependent for a connector. Could you explain the reason?

Reply:
I think the reason is `BatchTableSink` with `BoundedStream` still in blink planner.
We need unify `TableSource` and `TableSink` interface and  move them to api.

Reply:
Yes, we need blink planner for `BatchTableSink ` at the moment

Reply:
I'm afraid the source/sink interfaces are still under discussion and the interfaces which blink planner currently used are highly possible not the final solution.

Reply:
Thanks for the explanation. So the suggestion is we should instead implement `BatchTableSink` in flink-table-api-java-bridge, right?

Reply:
I think it's also not a good idea since we are deprecating `DataSet` now. How about:
1. Decouple the implementation details for this functionality and first merge the parts which unrelated to TableSink interface, like provide an OutputFormat and test it independently.
2. Provide the TableSink implementation after we decide and merge the TableSink interface into `common` or `api` package, and add some integration tests with it.